samples/BehaviorsTestApplication/Views/Pages/DragAndDropView.axaml

```diff
@@ -3,6 +3,7 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
              xmlns:vm="using:BehaviorsTestApplication.ViewModels"
+             xmlns:b="using:BehaviorsTestApplication.Behaviors"
```

Comment:
Not needed should work without prefix https://github.com/wieslawsoltes/Xaml.Behaviors/blob/master/samples/BehaviorsTestApplication/Properties/AssemblyInfo.cs#L6

Reply:
Done